### DISCUSSION TITLE
Lollipop no longer supported

### DISCUSSION THREAD

**Discussion by r-a-y** [ungoogled-software/ungoogled-chromium-android#109] [General]
As of v96, Chromium no longer supports Lollipop. See https://groups.google.com/a/chromium.org/g/chromium-dev/c/2MwR9KqwY9I/m/zpJ-fksqBwAJ.

Would be nice if Lollipop could still be supported if possible, but understand if resources are tight.

**Comment by wchen342** [comment]
I think it is not a hard limitation? You can probably still install, but some functionalities will not work correctly.

**Comment by r-a-y** [comment]
Just tried installing on a Lollipop device.

The following dialog prompt displays:

```
Parse error

There was a problem parsing the package.
```

So it appears to be a hard limitation.

**Reply by wchen342** [reply]
I was trying to find where the limit is but hadn't been able to. If I can find a simple way to revert it I will make a version otherwise there is not much I can do I think.

**Reply by r-a-y** [reply]
Maybe this ticket will help in finding the specific commits? https://bugs.chromium.org/p/chromium/issues/detail?id=1265456

**Reply by wchen342** [reply]
Unfortunately the issue is more a follow-up to the change that cleans up the code rather than the original change.
